app/src/main/java/fr/neamar/kiss/SettingsActivity.java

```diff
@@ -157,7 +157,7 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
         if (requireRestartSettings.contains(key)) {
             requireFullRestart = true;
 
-            if (key.equalsIgnoreCase("theme")) {
+            if (key.equalsIgnoreCase("dark-theme") || key.equalsIgnoreCase("transparency")) {
```

Comment:
This is actually the only potential issue I see, since picking your theme now requires up to two restarts for no valid reason.